spec/unit/checker_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'checker'
+
+RSpec.describe SlackApi::Checker do
+  it "requests and answer when the type is an event callback" do
+    checker = SlackApi::Checker.new(slack_data(type: "event_callback"))
+
+    expect(checker.require_answer?).to eq(true)
```

Comment:
minor detail, but how about `requires_answer` (with an s), so it reads even nicer?

Reply:
Little big details <3